test/scrabble_test.rb

```diff
@@ -1,13 +1,26 @@
 gem 'minitest'
-require_relative '../lib/scrabble'
+require './lib/scrabble'
 require 'minitest/autorun'
 require 'minitest/pride'
-require 'pry'
+require './lib/scrabble'
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
+
```

Comment:
Make sure you're watching your spacing in tests; setup should go at the top, assertion(s) at the bottom & a space goes between them (if both are present).